Requesting Signoff for Security & Locks

MTurpeau opened the pull request:
The Security & Locks subsystem

MTurpeau committed:
Create Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

JesseTNRoberts commented on Documentation/Signoffs/Security_Locks.md:
```diff
@@ -0,0 +1,58 @@
+
+**• The "state" of the locks must be known at all times.**
+	
+**• The labinet doors must remain lock in the case of power outage/blackout.**
```
correct typo

JesseTNRoberts commented on Documentation/Signoffs/Security_Locks.md:
```diff
@@ -0,0 +1,58 @@
+This subsystem's role in our design is to secure both the stored devices and internal components of the machine. The main contributor to the security system will be the cabinet doors holding the devices. These cabinet door require digitally controlled locks, allowing access to verified users, while locking out those who would attempt to forcefully steal from the machine.
+
+
+# Constraints
```
Digital is not enough to go on for a design. What are the communication constraints?

JesseTNRoberts replied:
Previous comments still need addressing.

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

JesseTNRoberts reviewed (commented):
The schematic is not specific and buildable by a 3rd party. Actually, the electrical schematic is insufficient to the point that this would not count as a valid signoff submission. But also, where is the 3d model showing how this lock will mount?

Where is the safety system being addressed for this project? None of this meets safety requirements. Which is ok as this seems to be just the access control circuitry. But safety needs addressing.

MTurpeau committed:
Update Security_Locks.md

Minor edits, still working on 2d and 3d schematics.

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

JesseTNRoberts commented on Documentation/Signoffs/Security_Locks.md:
```diff
@@ -0,0 +1,92 @@
+
+# Constraints
+
+**• The locks must be automated and controlled by the device.**
```
these need to all be precise and measurable.

JesseTNRoberts reviewed (commented):
This signoff is not structured properly. Analysis is before your design, this is incorrect (I need to know what you are analyzing first).

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

MTurpeau committed:
Update Security_Locks.md

Updated 3D model.
Re-added Inductance Sensors.
-Adjusted BOM
Slightly adjusted constraints.
Constraints still need work.

JesseTNRoberts commented on Documentation/Signoffs/Security_Locks.md:
```diff
@@ -0,0 +1,118 @@
+
+The proximity sensors will be installed into the sides of the opening of the cabinet doors. They will be directly aligned with the solenoid locks.
+
+### Sensor Schematic
```
Create a combined schematic for these items and show everything in that one schematic. Also, why are these connected to ground rather than neutral?

MTurpeau committed:
Update Security_Locks.md
